ztests/suite/zar/index-missing.yaml

```diff
@@ -0,0 +1,21 @@
+script: |
+  mkdir logs
+  zar import -R ./logs babble.tzng
+  echo ===
+  zar find -R ./logs v=10 # no such index
+  echo ===
+  zar find -R ./logs -Q v=10 && echo "success" # no such index
```

Comment:
```suggestion
  zar import -R logs babble.tzng
  echo ===
  zar find -R logs v=10 # no such index
  echo ===
  zar find -R logs -Q v=10 && echo success # no such index
```

Reply:
I'm leaving the `./logs` in as it matches the other zar yaml tests, and can change all of them at once.